tests/api/test_BitMask.py

```diff
@@ -150,13 +150,41 @@ def test_subset_a_list(self):
                 mask.set_kth_bit(index)
 
         all_ones = mask.masked_select_from_list(list_)
-        mask.x = mask.x ^ (2 ** n - 1) # inverse
+        mask.invert(n)
         all_zeros = mask.masked_select_from_list(list_)
         self.assertGreater(len(all_ones), 0)
         self.assertGreater(len(all_zeros), 0)
         self.assertTrue(all([item_ > 0 for item_ in all_ones]))
         self.assertTrue(all([item_ == 0 for item_ in all_zeros]))
 
+
+    def test_masked_select_from_list_example(self):
+        list_ = [1, 2, 3, 4, 5, 6]
+        mask = BitMask.from_bin('0b001101') # expected result is [1, 3, 4]
+        selected = mask.masked_select_from_list(list_)
+        self.assertListEqual(selected, [1, 3, 4])
+
+
+    def test_invert(self):
+        # get random bitstring
+        length = 10
+        bitstring = self.random_bitstring(10)
+ 
+        #get inverse
+        mask = BitMask.from_bin(bitstring)
+        mask.invert(length)
+        inverted = mask.to_bin()
+
+        # remove 0b
+        inverted = inverted[2:]
+        bitstring = bitstring[2:]
+        for i in range(min(len(bitstring), len(inverted))):
```

Comment:
is there a scenario where one string could be shorter than the other? like a '11110000' turning in to a '1111' when inverting? (I had that in the typescript side) If so, we should ensure that the same length pops out again

Reply:
as discussed privately this isnt a real issue and only apparent when using the string form.